package.json

```diff
@@ -41,6 +41,6 @@
     "nyc": "^15.0.0"
   },
   "engines": {
```

Comment:
Maybe also bump the stylus version, to be sure that the 0.54.8 is picked up by npm (no warning with node 14)

Reply:
updated